pkg/test/framework/features/features.yaml

```diff
@@ -61,15 +61,51 @@ features:
     user:
     ingress:
       mtls:
+        server-keyrotation:
+        generic-compoundrotation:
+        nongeneric-compoundrotation:
+        gateway:
+          invalid-secret:
+          valid-secret:
+          K8sca:
       tls:
+        secretrotation:
```

Comment:
```        secretrotation:
        gateway:
          invalid-secret:
```
These are test cases not features. Please leave them in the tests don't add into this file.